localstack-core/localstack/services/lambda_/event_source_mapping/pollers/stream_poller.py

```diff
@@ -326,3 +347,12 @@ def max_retries_exceeded(self, attempts: int) -> bool:
         if maximum_retry_attempts == -1:
             return False
         return attempts > maximum_retry_attempts
+
+    def bisect_events(
```

Comment:
nit (for follow up): a short docstring clarifying the intent could clarify the intent and make it easier to related to the ESM automatic bisect functionality.

Reply:
Done!